x-pack/plugins/endpoint/server/routes/resolver/tree.ts

```diff
@@ -4,16 +4,23 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
-import _ from 'lodash';
 import { schema } from '@kbn/config-schema';
 import { RequestHandler, Logger } from 'kibana/server';
-import { extractParentEntityID } from './utils/normalize';
-import { LifecycleQuery } from './queries/lifecycle';
-import { ResolverEvent } from '../../../common/types';
+import { Fetcher } from './utils/fetch';
+import { Tree } from './utils/tree';
 import { IndexPatternRetriever } from '../../index_pattern';
 
-interface LifecycleQueryParams {
+interface TreeQueryParams {
+  // the rough approximation of how many children you can request per process, per level
+  // really this only matters for the first level of children, every other level exponentially
+  // increases this value
```

Comment:
❔ Seems like this was pasted from the other place? Need them both or should one comment be tweaked?